### PR TITLE
Fix aside element for better scrolling on medium screens

### PR DESCRIPTION
**Pull request by K3ndev**
Added CSS position and height to aside element to improved usability and accessibility while scrolling, specifically when the screen width is 501-1023 dimension.

<details>
<summary>Before</summary>

https://user-images.githubusercontent.com/91131474/234581978-27712ffe-9fdc-42e3-8821-c9b4045bb1d5.mp4

</details>

<details>
<summary>After</summary>

https://user-images.githubusercontent.com/91131474/234582045-cd89b03a-c34e-4916-b3d2-da71b8d9e37f.mp4

</details>

### CODE REVIEW

File: src/layouts/MainLayout.vue [hunk 1]
```diff
@@ -224,3 +224,10 @@ export default {
   margin-right: 0;
 }
 </style>
+
+<style global>
```
Comment (jofftiquez) [inline src/layouts/MainLayout.vue:228]: Hi, please avoid declaring global styles. You can add this to the existing style. Thank you.

Addition: You can add a specific id to the nav component for this and target that.

**Review by jofftiquez — CHANGES_REQUESTED** [review]

File: src/layouts/MainLayout.vue (hunk 1 above)
Comment (K3ndev) [reply src/layouts/MainLayout.vue:228]: Hi i tried the solution you suggested but it didn't work, here's the code I used: 
```
---
  q-drawer(
    v-if="isMobile"
    v-model="drawerLeft"
    side="left"
    class="bg-primary fixed-aside"
    bordered
    :width="300"
    :breakpoint="500"
  )
---
</style scoped>
.fixed-aside{
  position: fixed !important;
  height: 100% !important;
}

.q-drawer{
  position: fixed !important;
  height: 100% !important;
}

aside{
  position: fixed !important;
  height: 100% !important;
}
</style>
```

## behavior 
- I noticed that when I add the fixed-aside class to the q-drawer element, it only applies the styles to the child element of the q-drawer component, not the q-drawer component itself. Additionally, the q-drawer component has its own default position of absolute.
- Do you have any suggestions? I'm not familiar with Quasar.

image

Comment (K3ndev) [reply src/layouts/MainLayout.vue:228]: Hi i tried the solution you suggested but it didn't work, here's the code I used: 
```
---
  q-drawer(
    v-if="isMobile"
    v-model="drawerLeft"
    side="left"
    class="bg-primary fixed-aside"
    bordered
    :width="300"
    :breakpoint="500"
  )
---
</style scoped>
.fixed-aside{
  position: fixed !important;
  height: 100% !important;
}

.q-drawer{
  position: fixed !important;
  height: 100% !important;
}

aside{
  position: fixed !important;
  height: 100% !important;
}
</style>
```

## behavior 
- I noticed that when I add the fixed-aside class to the q-drawer element, it only applies the styles to the child element of the q-drawer component, not the q-drawer component itself. Additionally, the q-drawer component has its own default position of absolute.
-  I tried adding a unique ID to the q-drawer component, but unfortunately, it didn't work. 
- I also did some research on how to target the parent element of the q-drawer component, but couldn't find a solution. 

Do you have any other suggestions on how to resolve this issue?

image

Comment (K3ndev) [reply src/layouts/MainLayout.vue:228]: Do you have any other suggestions on how to resolve this issue, or do you think we should close this PR for now? 

Thank you.

Comment (jofftiquez) [reply src/layouts/MainLayout.vue:228]: Please put this on hold for now I will review and test the solution. I am also not sure why it doesn't work at the moment. Thank you

**Comment by jofftiquez** [comment]
Outdated based on the new major changes. Closing this now. Thanks

**Pull request closed without merging**